### PR TITLE
matrix-commander: 6.0.1 -> 7.2.0

### PR DESCRIPTION
**Pull request by mcnesium**
matrix-commander: 6.0.1 -> 7.2.0

### CODE REVIEW

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Please make the commit message fit our [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions).

Diff looks good!

**Comment by matthiasbeyer** [comment]
You did change the title of the PR, but not the commit message.

File: pkgs/applications/networking/instant-messengers/matrix-commander/default.nix [hunk 1]
```diff
@@ -12,17 +12,18 @@
...
   pname = "matrix-commander";
-  version = "6.0.1";
+  version = "7.2.0";
 
   src = fetchFromGitHub {
     owner = "8go";
     repo = "matrix-commander";
     rev = "v${version}";
-    sha256 = "sha256-NSoMGUQjy4TQXdzZcQfO2rUQDsuSzQnoGDpqFiLQHVQ=";
+    sha256 = "sha256-qL6ARkAWu0FEuYK2e9Z9hMSfK4TW0kGgoIFUfJ8Dgwk=";
```
Comment (AndersonTorres) [inline pkgs/applications/networking/instant-messengers/matrix-commander/default.nix:26]:
```suggestion
    hash = "sha256-qL6ARkAWu0FEuYK2e9Z9hMSfK4TW0kGgoIFUfJ8Dgwk=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

**Review by seb314 — COMMENTED** [review]
diff looks good (I didn't test it though)
There still seem to be 2 separate commits (e.g. interactive rebase with reword for the first commit and fixup for the second should fix that, I think)

**Comment by mcnesium** [comment]
@seb314 sorry I only clicked what the UI here on Github asked me to. I have added another PR hopefully being fully compliant now.

**Comment by seb314** [comment]
Note: there's now new PR at https://github.com/NixOS/nixpkgs/pull/250528 with identical diff in a single commit

**Pull request closed without merging**